src/workloads/execution/BackgroundTTLDeletions.yml

```diff
@@ -0,0 +1,177 @@
+
+Document: &generatedDoc
+  expireDate: {^Now: {}}  # Instantly expire the document
+  y: &y {^RandomInt: {min: -1000, max: 1000}}
+  b: {^RandomString: {length: 15}}
+
+ActorTemplates:
+- TemplateName: WritesActorTemplate
+  Config:
+    Name: {^Parameter: {Name: "Name", Default: WritesActor}}
+    Type: CrudActor
+    Database: &db test
+    Threads: {^Parameter: {Name: "Threads", Default: 1}}
+    Phases:
+      OnlyActiveInPhases:
+        Active: {^Parameter: {Name: "ActivePhases", Default: [1]}}
+        NopInPhasesUpTo: *NumPhases
+        PhaseConfig:
+          Duration: 3 minutes
+          CollectionCount: *NumColls
+          Operations:
+          - OperationName: insertMany
+            OperationCommand:
+              Documents:  # Insert 50 documents at a time
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+
+
+Actors:
+- Name: Setup
+  Type: AdminCommand
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *NumPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: admin
+        Operations:
+        - OperationName: AdminCommand
+          OperationCommand:
+            setParameter: 1
+            ttlMonitorSleepSecs: 1
+        - OperationName: AdminCommand
+          OperationCommand:
+            setParameter: 1
+            storageEngineConcurrentWriteTransactions: 32
+        SleepAfter: 60 seconds  # Wait 60 seconds so that the new ttl sleep config is active.
+
+- Name: InsertData
```

Comment:
What is the reason for having this loading phase before the actual insert phases below?

Reply:
Sets up the TTL index mostly